pkg/operator/ceph/object/status_test.go

```diff
@@ -35,5 +36,36 @@ func TestBuildStatusInfo(t *testing.T) {
 
 	statusInfo := buildStatusInfo(cephObjectStore)
 	assert.NotEmpty(t, statusInfo["endpoint"])
+	assert.Empty(t, statusInfo["secureEndpoint"])
 	assert.Equal(t, "http://rook-ceph-rgw-my-store.rook-ceph:80", statusInfo["endpoint"])
+
+	// SecurePort enabled and Port disabled
+	cephObjectStore = &cephv1.CephObjectStore{
```

Comment:
Why not just re-using cephObjectStore and simply changing/re-setting values instead of overriding?

Reply:
Done.